x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/inference/action/GetInferenceModelAction.java

```diff
@@ -111,7 +111,7 @@ public void writeTo(StreamOutput out) throws IOException {
         @Override
         public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
             builder.startObject();
-            builder.startArray("models");
+            builder.startArray("endpoints");
             for (var model : models) {
                 if (model != null) {
                     model.toFilteredXContent(builder, params);
```

Comment:
it would be nice to update these variables as well:
```suggestion
            for (var endpoint : endpoints) {
                if (endpoint != null) {
                    endpoint.toFilteredXContent(builder, params);
```